packages/base-driver/lib/basedriver/core.ts

```diff
@@ -106,14 +98,10 @@ export class DriverCore<const C extends Constraints, Settings extends StringReco
 
   protocol?: Protocol;
 
-  bidiEventSubs: Record<string, string[]>;
-
-  doesSupportBidi: boolean;
-
-  bidiCommands: BidiModuleMap = BIDI_COMMANDS as BidiModuleMap;
-
   constructor(opts: InitialOpts = <InitialOpts>{}, shouldValidateCaps = true) {
-    this._log = logger.getLogger(helpers.generateDriverLogPrefix(this as Core<C>));
+    super();
+    this._log = this.log; // TODO: remove references to this._log and use this.log instead
```

Comment:
is this change related to the one in PR title? would it be better to split them?